app/styleguide/wskComponentHandler.js

```diff
@@ -151,5 +150,16 @@ var componentHandler = (function() {
 window.addEventListener('load', function() {
   'use strict';
 
-  componentHandler.upgradeAllRegistered();
+  /**
+   * Performs a "Cutting the mustard" test. If the browser supports the features
+   * tested, adds a wsk-js class to the <html> element. It then upgrades all WSK
+   * components requiring JavaScript.
+   */
+  if ('classList' in document.createElement('div') && 'querySelector' in document && 'addEventListener' in window && Array.prototype.forEach) {
+    document.documentElement.className += 'wsk-js';
+    componentHandler.upgradeAllRegistered();
+  } else {
+    componentHandler.register = function () { /*noop*/ };
```

Comment:
I wonder if having the `/*noop*/` comment in makes sense. I mean, an empty function is something we all know right?